users/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.core import validators
+from django.contrib.auth.models import AbstractBaseUser, UserManager
+
+
+class User(AbstractBaseUser):
+
```

Comment:
no need in empty line here. Consider adding linting and type checking configs in `setup.cfg`